Assets/World/Buildings/Lumberjack/Lumberjack.gd

```diff
@@ -167,22 +167,21 @@ func add_resource() -> void:
 func remove_resource() -> void:
 	pass # TODO
 
-func _on_input(event: InputEvent):
-	super(event)
+func _on_camera_rotate_left() -> void:
+	if null in [resource_overlay, resource_overlay2]: return
```

Comment:
Missing `super` call (rotation won't work otherwise).
```suggestion
func _on_camera_rotate_left() -> void:
    super()

	if null in [resource_overlay, resource_overlay2]: return
```